src/test-run/index.js

```diff
@@ -438,7 +440,7 @@ ServiceMessages[CLIENT_MESSAGES.waitForFileDownload] = function (msg) {
 ServiceMessages[CLIENT_MESSAGES.showDebuggerMessage] = function (msg) {
     this.debugLog.driverMessage(msg);
 
-    showDebuggerMessage(this.currentDriverTask.callsite, this.browserConnection.userAgent);
+    debuggerMessage.show(this.id, this.browserConnection.userAgent, this.currentDriverTask.callsite);
```

Comment:
It should be named `debugLogger`.
`show` -> `showBreakpoint`,
`remove ` -> `hideBreakpoint`
`id` -> `testRunId`